tests/integration/startup.test.js

```diff
@@ -111,6 +111,10 @@ describe('daemon startup', function () {
                 setTimeout(done, 2000);
             });
 
+            afterEach('kill daemon', function () {
+                this.daemon.kill();
```

Comment:
Not quite safe: Something prior to setting `this.daemon` could fail. Need to check if it exists first.

Reply:
Added undefined guard